openedx/stanford/cms/djangoapps/contentstore/views/course.py

```diff
@@ -0,0 +1,86 @@
+from smtplib import SMTPException
+
+from django.conf import settings
+from django.core.exceptions import PermissionDenied
+from django.core.urlresolvers import reverse
+from django.http import HttpResponse, HttpResponseBadRequest
+from django.views.decorators.http import require_http_methods
+
+from opaque_keys.edx.keys import CourseKey
+
+from edxmako.shortcuts import render_to_string
+from microsite_configuration import microsite
+from student.auth import has_studio_read_access
+from util.date_utils import get_default_time_display
+from util.keyword_substitution import get_keywords_supported
+from util.keyword_substitution import substitute_keywords_with_data
+from xmodule.modulestore.django import modulestore
+
+
+# TODO: Stop redefining here, instead move CMS original to common
+# Avoid circular dependency
+def get_course_and_check_access(course_key, user, depth=0):
```

Comment:
I had to (at least for now) duplicate this; it does exist elsewhere in the platform.

Otherwise, there are circular dependency issues.